sass/components/_btn.scss

```diff
@@ -1,18 +1,19 @@
+
+//Style buttonów zmieniłem na razie pod to co wskazuje projekt, pewnie będzie trzeba zrobić refactor w przyszłości - Tomek
+
 .btn {
-  // overflow: hidden;
-  // height: 50px;
-  // width: 200px;
-  // border-radius: 15px;
-  // border: none;
-  // background-image: linear-gradient(270deg, #ffc53d 14%, #ff9a3d 68.5%);
-  // text-align: center;
-  // color: $color-font--on-brand-bgc;
-  // font-family: $font-family--primary;
-  // font-size: 18px;
-  // font-style: normal;
-  // font-weight: 700;
-  // line-height: 50px;
-  // letter-spacing: 0em;
+  height: 100px;
+  width: 575px;
```

Comment:
Jak odpalam apkę to zdaje się trochę za duży jak na wymiary, które przyjęliśmy.

Reply:
a tak tak, to wziąłem z Figmy jak jeszcze myślałem, że robimy najpierw na desktopy :P

Reply:
Robimy na desktopy, ale te mniejsze :)